cpp/tests/io/json_test.cpp

```diff
@@ -648,7 +649,7 @@ TEST_F(JsonReaderTest, NoDataFile) {
   EXPECT_EQ(0, view.num_columns());
 }
 */
-
+/*
 TEST_F(JsonReaderTest, ArrowFileSource)
```

Comment:
Why do you comment out this?

Reply:
oops. I had commented out all the tests except the one I was working on so it made it easier to use GDB against that test. I meant to uncomment them before committing. I will change that in the next commit.